### PR TITLE
config/anycable.yml 

### PR DESCRIPTION
**Pull request by alekseyl**
Config/anycable.yml with additional comments inside added to anycable-rails generators.

### CODE REVIEW

File: lib/generators/anycable/templates/anycable.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+# MUST BE in sync with ws-server setting  -redis_channel=__anycable__
+# redis_channel: "__anycable__",
+
+# log_file: nil,
+# log_level: :info,
+# log_grpc: false,
+# debug: false # Shortcut to enable GRPC logging and debug level
+
+
+production:
```
Comment (palkan) [inline lib/generators/anycable/templates/anycable.yml]: Let's add comments about the corresponding ENV variables, i.e.:

```yml
# Every setting could be overwritten through 
# the corresponding environment variable (see below)
production:
  # ENV: ANYCABLE_RPC_HOST
  rpc_host: "localhost:50051"
  # ...
``

File: lib/generators/anycable/templates/anycable.yml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  redis_url: "redis://localhost:6379/1"
+  redis_sentinels: []
+  redis_channel: "__anycable__"
+  log_file: nil
+  log_level: info
+  log_grpc: false
+  debug: false # Shortcut to enable GRPC logging and debug level
+
+development:
+  rpc_host: "localhost:50051"
```
Comment (palkan) [inline lib/generators/anycable/templates/anycable.yml]: What about extracting the commons into a `default` key and use inheritance?

File: lib/generators/anycable/templates/anycable.yml (hunk 1 above)
Comment (alekseyl) [reply lib/generators/anycable/templates/anycable.yml]: fixed

File: lib/generators/anycable/templates/anycable.yml (hunk 2 above)
Comment (alekseyl) [reply lib/generators/anycable/templates/anycable.yml]: fixed

File: lib/generators/anycable/templates/anycable.yml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+production:
+  <<: *default
+  redis_url: "redis://localhost:6379/1"
+  redis_sentinels: []
+  redis_channel: "__anycable__"
+
+development:
+  <<: *default
+  redis_url: "redis://localhost:6379/2"
+  redis_sentinels: []
```
Comment (palkan) [inline lib/generators/anycable/templates/anycable.yml]: Let's remove this setting from development; it doesn't make sense

Comment (alekseyl) [reply lib/generators/anycable/templates/anycable.yml]: Done with all of them except for redis_url

**Comment by palkan** [comment]
👍 
Let's add a change log item (under "Unreleased" section) and we're done.

**Comment by alekseyl** [comment]
Добавил :)

**Comment by palkan** [comment]
Спасибо!

**Pull request merged**